chunky/src/java/se/llbit/chunky/chunk/biome/BiomeData2d.java

```diff
@@ -46,7 +47,7 @@ public static void loadBiomeDataByteArray(Tag chunkData, BiomeData2d biomeData,
 
   public static void loadBiomeDataIntArray(Tag chunkData, BiomeData2d biomeData, BiomePalette biomePalette) {
```

Comment:
Same here, ie. let the biome data classes only contain the _data_ and add per world format factories to create them?